modules/gapi/src/api/media.cpp

```diff
@@ -0,0 +1,42 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#include "precomp.hpp"
```

Comment:
relative paths should be used (opencv_world build with PCH is broken - `precomp.hpp` from other module is used)

Reply:
Will fix in a follow-up MR, thanks

Reply:
Wait.. We've been never using relative paths for `precomp.hpp`:

```
grep -Rn "precomp.h" .
./api/ft_render.cpp:7:#include "precomp.hpp"
./api/garray.cpp:8:#include "precomp.hpp"
./api/gcall.cpp:8:#include "precomp.hpp"
./api/gcomputation.cpp:8:#include "precomp.hpp"
./api/gframe.cpp:8:#include "precomp.hpp"
./api/gframe.cpp~:8:#include "precomp.hpp"
./api/ginfer.cpp:8:#include "precomp.hpp"
./api/gkernel.cpp:8:#include "precomp.hpp"
./api/gmat.cpp:8:#include "precomp.hpp"
./api/gnode.cpp:8:#include "precomp.hpp"
./api/gopaque.cpp:8:#include "precomp.hpp"
./api/gorigin.cpp:8:#include "precomp.hpp"
./api/gproto.cpp:8:#include "precomp.hpp"
./api/gscalar.cpp:8:#include "precomp.hpp"
./api/kernels_core.cpp:8:#include "precomp.hpp"
./api/kernels_imgproc.cpp:8:#include "precomp.hpp"
./api/kernels_nnparsers.cpp:8:#include "precomp.hpp"
./api/kernels_video.cpp:8:#include "precomp.hpp"
./api/operators.cpp:8:#include "precomp.hpp"
./api/render.cpp:1:#include "precomp.hpp"
./backends/common/gcompoundbackend.cpp:8:#include "precomp.hpp"
./backends/common/gcompoundkernel.cpp:8:#include "precomp.hpp"
./backends/cpu/gcpubackend.cpp:8:#include "precomp.hpp"
./backends/cpu/gcpucore.cpp:8:#include "precomp.hpp"
./backends/cpu/gcpuimgproc.cpp:8:#include "precomp.hpp"
./backends/cpu/gcpukernel.cpp:8:#include "precomp.hpp"
./backends/cpu/gcpuvideo.cpp:8:#include "precomp.hpp"
./backends/fluid/gfluidbackend.cpp:8:#include "precomp.hpp"
./backends/fluid/gfluidbuffer.cpp:8:#include "precomp.hpp"
./backends/fluid/gfluidcore.cpp:9:#include "precomp.hpp"
./backends/fluid/gfluidimgproc.cpp:9:#include "precomp.hpp"
./backends/ie/giebackend.cpp:7:#include "precomp.hpp"
./backends/ocl/goclbackend.cpp:8:#include "precomp.hpp"
./backends/ocl/goclcore.cpp:8:#include "precomp.hpp"
./backends/ocl/goclimgproc.cpp:8:#include "precomp.hpp"
./backends/openvx/govxbackend.cpp~:9:#include "precomp.hpp"
./backends/plaidml/gplaidmlbackend.cpp:10:#include "precomp.hpp"
./backends/plaidml/gplaidmlcore.cpp:8:#include "precomp.hpp"
./backends/render/grenderocvbackend.cpp:7:#include "precomp.hpp"
./compiler/gcompiled.cpp:8:#include "precomp.hpp"
./compiler/gcompiler.cpp:8:#include "precomp.hpp"
./compiler/gislandmodel.cpp:8:#include "precomp.hpp"
./compiler/gmodel.cpp:8:#include "precomp.hpp"
./compiler/gmodelbuilder.cpp:14:#include "precomp.hpp"
./compiler/gstreaming.cpp:8:#include "precomp.hpp"
./compiler/passes/dump_dot.cpp:8:#include "precomp.hpp"
./compiler/passes/exec.cpp:8:#include "precomp.hpp"
./compiler/passes/helpers.cpp:8:#include "precomp.hpp"
./compiler/passes/intrin.cpp~:8:#include "precomp.hpp"
./compiler/passes/islands.cpp:8:#include "precomp.hpp"
./compiler/passes/kernels.cpp:8:#include "precomp.hpp"
./compiler/passes/meta.cpp:8:#include "precomp.hpp"
./compiler/passes/streaming.cpp:7:#include "precomp.hpp"
./compiler/passes/transformations.cpp:7:#include "precomp.hpp"
./executor/gexecutor.cpp:8:#include "precomp.hpp"
./executor/gstreamingexecutor.cpp:7:#include "precomp.hpp"
```

Is this something new we should follow now?